.ci/azure/linux.yml

```diff
@@ -191,14 +194,27 @@ jobs:
       export DATA_PATH=$(MODELS_PATH)
       export MODELS_PATH=$(MODELS_PATH)
       export LD_LIBRARY_PATH=$(BIN_DIR)/lib
-      export PYTHONPATH=$(BIN_DIR)/lib/python_api/python3.6
+      export PYTHONPATH=$(PYTHONPATH)
       env
       cd $(REPO_DIR)/inference-engine/ie_bridges/python/tests
       pytest pytest --junitxml=TEST-PythonAPI.xml
     displayName: 'Python API Tests'
     continueOnError: false
     enabled: false
 
+  - script: |
+      export MO_ROOT=$(MO_ROOT)
```

Comment:
should not be used

Reply:
It's a specific requirement for the tests. To remove dependency, need to update tests too